CHANGELOG.md

```diff
@@ -3,16 +3,28 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 2.2.0 - Dogmatix [XXXX-XX-XX]
+## 2.2.0dev - Dogmatix [XXXX-XX-XX]
 
 ### `Added`
 
+- Two new parameters `mbuffer_mem` and `samtools_sort_threads` to control resources given to mbuffer and samtools sort in the bwameme module [#570](https://github.com/nf-core/raredisease/pull/570)
+
 ### `Changed`
 
+- `readcount_intervals` parameter is now mandatory for running germlinecnvcaller. [#570](https://github.com/nf-core/raredisease/pull/570)
```

Comment:
Could you add to the documentation that `readcount_intervals` and the model can be generated using `nf-core/createpanelrefs` and that the intervals needs to be the same as the ones used to generate the model.